### PR TITLE
Add plugin development doc

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
This PR adds documentation about local plugin development

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [X] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [X] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/plugin-development/frontend-development.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-### Admin Lifecycle
+## Admin panel
 
-The admin package has the following lifecycle.
+Strapi's admin panel and plugins system aim to be an easy and powerful way to create new features.
 
-1. Retrieve all the installed plugin and and store them into the main redux store
-2. Load until all the plugin emit the event `isReady`
-3. Runtime
+The admin panel is a [React](https://facebook.github.io/react/) application which can embed other React applications. These other React applications are the `admin` parts of each Strapi's plugins.
```
Comment (derrickmehaffy) [inline docs/3.0.0-beta.x/plugin-development/frontend-development.md]: Duplicate sentence from line 5?

Comment (soupette) [reply docs/3.0.0-beta.x/plugin-development/frontend-development.md]: Yes I'll remove it!

**Comment by federgl** [comment]
Hello guys! Just want to know when this is gonna be merged, so we can see it live on the strapi.io docs site, and use this of course too! Thanks!

**Comment by lauriejim** [comment]
I'm currently deploying this doc

**Pull request merged**